src/test/java/me/itzg/helpers/assertcmd/FileExistsTest.java

```diff
@@ -53,4 +53,40 @@ void failsWhenSomeMissing(@TempDir Path tempDir) throws Exception {
         "fileB does not exist"
     );
   }
-}
\ No newline at end of file
+
+  @Test
+  void passesWhenGlobFindsAllFiles(@TempDir Path tempDir) throws Exception {
+    Files.createFile(tempDir.resolve("file1"));
+    Files.createFile(tempDir.resolve("file2"));
+
+    final String errOut = tapSystemErr(() -> {
+      int exitCode = new CommandLine(new FileExists())
+          .execute(
+              String.format("%s/file*", tempDir)
```

Comment:
This might fail on Windows; however, it seems like Windows is now forward-slash tolerant. If it is sensitive the fix would be:
```suggestion
              String.format("%s%sfile*", tempDir, File.separator)
```

Reply:
I was running it in windows and it  wasn't failing.